sklearn/tests/test_public_functions.py

```diff
@@ -103,6 +103,7 @@ def _check_function_param_validation(
     "sklearn.metrics.mean_absolute_error",
     "sklearn.metrics.roc_curve",
     "sklearn.metrics.zero_one_loss",
+    "sklearn.metrics.confusion_matrix",
```

Comment:
can you put it at the beginning of the `sklearn.metrics` to be in alphabetic order?